### PR TITLE
SRVCOM-1430: Updated 1.16.0 release notes

### PR DESCRIPTION
**Pull request by abrennan89**
Applies for OCP 4.6+

Jira: https://issues.redhat.com/browse/SRVCOM-1430

Direct preview links:
- release notes: https://deploy-preview-34567--osdocs.netlify.app/openshift-enterprise/latest/serverless/serverless-release-notes.html#serverless-rn-1-16-0_serverless-release-notes
- metrics notes:
  - https://deploy-preview-34567--osdocs.netlify.app/openshift-enterprise/latest/serverless/admin_guide/serverless-admin-metrics.html#prerequisites_serverless-admin-metrics
  - https://deploy-preview-34567--osdocs.netlify.app/openshift-enterprise/latest/serverless/knative_serving/serverless-serving-metrics.html#prerequisites_serverless-metrics

### CODE REVIEW

File: modules/serverless-rn-1-16-0.adoc [hunk 1]
```diff
@@ -12,13 +12,19 @@
...
 
-// Add a note about the following to the mTLS docs when they're ready to merge
+* The AMQ Streams Operator might prevent the installation or upgrade of the {ServerlessOperatorName}. If this happens, the following error is thrown by Operator Lifecycle Manager (OLM):
++
+[source,terminal]
+----
+WARNING: found multiple channel heads: [amqstreams.v1.7.2 amqstreams.v1.6.2], please check the `replaces`/`skipRange` fields of the operator bundles.
+----
++
+Uninstalling the AMQ Streams Operator temporarily fixes this issue.
```
Comment (alanfx) [inline modules/serverless-rn-1-16-0.adoc:25]: Hey @abrennan89 you might also want to say that once serverless has been upgraded, you can reinstall AMQ Streams?

Comment (mgencur) [reply modules/serverless-rn-1-16-0.adoc:25]: Isn't that covered by the word "temporarily"? We could also say it more explicitly.

Comment (abrennan89) [reply modules/serverless-rn-1-16-0.adoc:25]: Yeah, I'll maybe remove "temporarily" and just say you can fix it by removing AMQ Streams but you can reinstall it after

**Comment by mgencur** [comment]
QE Approved

**Review by skrthomas — APPROVED** [review]
LGTM!

**Pull request merged**